src/content/docs/journey-performance/slo-tutorial.mdx

```diff
@@ -0,0 +1,296 @@
+        * Where `{cumulativeLayoutShift}` is a pre-set value. To provide a good user experience, your site should strive to have a CLS score of 0.1 or less. A CLS score of 0.25 or more is considered a poor user experience.
+
+        If you’ve decided to create separate SLIs to track CLS in desktop and mobile devices separately when you defined the valid events query, add this clause at the end of the field:
+
+        * `and deviceType = 'Mobile'`
+        * `and deviceType = 'Desktop'`
+
+        To determine a realistic number to select for `{cumulativeLayoutShift}` in your environment, one typical practice is to select the 75th percentile of page loads for the last 7 or 15 days, segmented across mobile and desktop devices. Find it by using the query builder:
+
+        ```sql  
+        SELECT percentile(cumulativeLayoutShift, 95) FROM PageViewTiming WHERE  entityGuid = '{entityGuid}' since 7 days ago limit max facet deviceType
+        ```
+    </Collapser>
+    </CollapserGroup>
+
+    <Callout variant="tip">
+    Your organization should define SLOs and SLIs based on your specific needs, your user's expectations, and resources available. After completing this tutorial, we recommnd you [learn more about how to define granular custom service levels](/docs/service-level-management/create-slm/#define-user-exp).
+    </Callout>
+    
+    </Step>
+
+    <Step>
+    
+
+    <img
+    title="SLI ui"
+    alt="An image of the service level maangement ui"
+    src={sonicSliUiOverview}
+    />
+
+    1. Navigate to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > Service levels management**. This UI shows all your service levels and allows you to define, monitor, and edit them. 
+    2. Select **+ Add a servic level** in the top right of the UI.
+    </Step>
+
+    <Step>
+    Choose the corresponding entity that you want to create a service level for. This could be an entire workload, a specific service, a synthetic monitor, or even a specific transaction. Once you've selected your entity, click **Continue** on the left side of the UI pane.
+    </Step>
+
+    <Step>
+
+     [UI IMAGE GOES HERE WITH EXAMPLE]   
+
+    Define the SLI you chose in step one in this pane. For example if I chose to define an SLI for browser app success, I would use the following queries:
+
+    **Query for valid events**:
+    ```
+    FROM: PageView
+    WHERE: entityGuid = '{entityGuid}'
+    ```
+
+    **Query for bad events**:
+    ```
+    FROM: JavaScriptError
+    WHERE: entityGuid = '{entityGuid}' AND firstErrorInSession IS true
+    ```
+
+    Select **Continue** in the left pane once you've confirmed your queries are correct.
+    </Step>
+
+</Steps>
+
+## Next steps
+
+Ready to learn more about what you can do with New Relic to improve your site performance? Check out our docs on service level management, and learn more about how you can use this feature to optimize your customer experience.
```

Comment:
```suggestion
Ready to learn more about what you can do with New Relic to improve your site performance? Check out our docs on service-level management, and learn more about how you can use this feature to optimize your customer experience.
```

Reply:
Should the SLM docs link somewhere in case they want to go directly? I'm actually not sure how we're handling links like that so I wanted to bring it up.

Reply:
We're re-working this section a bit, but good eye!